app/soundcloud.js

```diff
@@ -50,8 +50,9 @@ module.exports = class SoundCloud extends Events {
 
   isPlaying() {
     return new Promise(resolve => {
-      ipcMain.once('isPlaying', (_, isPlaying) => {
-        resolve(isPlaying)
+      ipcMain.once('isPlaying', (_, isPlaying, icon) => {
+        var values = [isPlaying, icon]
+        resolve(values)
```

Comment:
As this might grow with more values in the future and keeping track of many of them in an array is messy, I would prefer to pass in an object:

```js
resolve({isPlaying, icon})
```

..and of course adjust the receiving side of this.